AsynchronApp/AsynchronApp/StartRepair.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AsynchronApp
+{
+    public class StartRepair
+    {
+        public static async Task Start()
+        {
+            Console.WriteLine("Apartment renovation");
+            CreateDesignProject();
+
+            var wall = WallRepair();
+            var floor = FloorRepair();
+            Roof ceiling = CeilingRepair();
+            Wall _wall = await wall;
+            Floor _floor = await floor;
```

Comment:
variable names starting with should be given to private fields, but not local variables